From fb9729999081978bfb4dae8a1e732a948ce45860 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Mon, 30 Aug 2010 17:54:53 +0000
Subject: [PATCH 19/63] gettext tests: mark a test message as not needing translation
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This was the only message in t/t0200/* that didn't have a TRANSLATORS
comment, without it translators will waste time translating this
needlessly.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/t0200/test.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/t/t0200/test.c b/t/t0200/test.c
index ff15c2f..584d45c 100644
--- a/t/t0200/test.c
+++ b/t/t0200/test.c
@@ -1,6 +1,7 @@
 /* This is a phony C program that's only here to test xgettext message extraction */
 
 const char help[] =
+	/* TRANSLATORS: This is a test. You don't need to translate it. */
 	N_("See 'git help COMMAND' for more information on a specific command.");
 
 int main(void)
-- 
1.7.2.3.313.gcd15

